### PR TITLE
Support for custom Icons

### PR DESCRIPTION
**Pull request by edward-shen**
See #2.

### CODE REVIEW

**Comment by edward-shen** [comment]
Also, if possible, please rebase your fork to get rid of the merge conflict.

File: MMM-page-indicator.js [hunk 1]
```diff
@@ -4,55 +4,63 @@ Module.register("MMM-page-indicator", {
         activeBright: false,
         inactiveDimmed: true,
         inactiveHollow: true,
-        
+        pageIcons: [],
+        icon: ' fa-circle',
+        hollowIcon: " fa-circle-thin",
+        iconSize: ''
```
[the remark this answers is not in the document]
Comment (stephenjgray) [reply MMM-page-indicator.js]: It's not but that's a good idea, how about we open an improvement to cover the work nessecary for that and the comment below?

File: MMM-page-indicator.js [hunk 2]
```diff
@@ -4,55 +4,63 @@ Module.register("MMM-page-indicator", {
         activeBright: false,
         inactiveDimmed: true,
         inactiveHollow: true,
-        
+        pageIcons: [],
+        icon: ' fa-circle',
+        hollowIcon: " fa-circle-thin",
```
[the remark this answers is not in the document]
Comment (stephenjgray) [reply MMM-page-indicator.js]: Updated and committed to master

**Comment by edward-shen** [comment]
Thanks for documentation! You still haven't rebased on top of changes from edward-shen/master, so there are still merge conflicts.

Do the following:
```
git remote add upstream git@github.com:edward-shen/MMM-page-indicator.git
git fetch upstream
git rebase upstream/master
```

**Comment by stephenjgray** [comment]
I think we might be better off nuking this PR and adding my changes to a fresh branch

**Comment by edward-shen** [comment]
I'm inclined to agree.

If I were you, I'd make another branch that's equal to upstream/master, and then cherry pick your commits from your master onto the branch. Then make a PR to merge that branch with upstream/master.

I'll leave this PR open until you request another PR.

**Comment by stephenjgray** [comment]
Try now, if not I'll nuke this and start again.

**Comment by edward-shen** [comment]
You need to make a new branch and request a PR on that branch--this PR is based on your master branch, and will attempt to pull in 14 commits.

**Comment by edward-shen** [comment]
Closing due to age.

**Pull request closed without merging**